### PR TITLE
Feat/avoid boring reminders

### PR DESCRIPTION
**Pull request by simone3991**

### CODE REVIEW

File: src/Clockify/ClockifyMessageSource.cs [hunk 1]
```diff
@@ -27,18 +31,27 @@ public ClockifyMessageSource(IStringLocalizer<ClockifyMessageSource> localizer)
...
+
         public string RemindEntryFill => GetString(nameof(RemindEntryFill));
 
-        private string GetString(string name) => _localizer[name];
+        private string GetString(string name)
+        {
+            string? key = _localizer.GetAllStrings()
+                .Select(s => s.Name)
+                .Where(k => k.StartsWith(name))
+                .OrderBy(_ => Rnd.Next())
```
Comment (khaelys) [inline src/Clockify/ClockifyMessageSource.cs:52]: How does it works? It's not clear

**Review by khaelys — CHANGES_REQUESTED** [review]

File: src/Clockify/ClockifyMessageSource.cs (hunk 1 above)
Comment (simone3991) [reply src/Clockify/ClockifyMessageSource.cs:52]: I take all possible resource values that starts with the target key, and pick a random one

Comment (khaelys) [reply src/Clockify/ClockifyMessageSource.cs:52]: I am referring only to the order by clause

Comment (simone3991) [reply src/Clockify/ClockifyMessageSource.cs:52]: you order by an expression whose values are random => you shuffle

**Review by khaelys — APPROVED** [review]

**Pull request merged**